src/wasm.ts

```diff
@@ -74,3 +74,11 @@ export function getStr(wasm: Wasm, ptr: number): string {
     return str;
   }
 }
+
+export function getSQLiteError(wasm: Wasm, code?: number): SqliteError {
```

Comment:
Note to self: I feel like this should be the `SqliteError` constructor. Maybe a signature like
```typescript
class SqliteError extends Error {
  constructor(context: Wasm | {code: number, message: string});
}